config/locales/en/en.yml

```diff
@@ -48,6 +48,8 @@ en:
         name: "Household Name"
         community_id: "Community"
         unit_num: "Unit Number"
+        unit_suffix: "Unit Suffix"
```

Comment:
Don't think we need this, or the `unit_num` one anymore, since there is no form field for those attribs.